homeassistant/components/fastdotcom/sensor.py

```diff
@@ -37,13 +38,20 @@ class SpeedtestSensor(
     _attr_state_class = SensorStateClass.MEASUREMENT
     _attr_icon = "mdi:speedometer"
     _attr_should_poll = False
+    _attr_has_entity_name = True
 
     def __init__(
         self, entry_id: str, coordinator: FastdotcomDataUpdateCoordindator
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
         self._attr_unique_id = entry_id
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, self.coordinator.config_entry.entry_id)},
+            name=DEFAULT_NAME,
+            entry_type=DeviceEntryType.SERVICE,
+            configuration_url="https://www.fast.com/",
```

Comment:
Do we have a better link to the right place instead of just generically to their website?

Reply:
Not really. Only a few news pages mentioning Fast.com, that's a service offered by Netflix. No much to it.